CMakeLists.txt

```diff
@@ -11,8 +11,28 @@ endif()
 set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
 
+set(USE_LIBUV ON)
```

Comment:
All the codes here only related to ``gloo``, so I think these compile options should be added in ``CMakeLists.txt`` under ``cpp/collective/gloo``. You can try it and see whether this way works.

Reply:
USE_LIBUV should be set to ON before we build third_party `gloo`, but cpp is built after it